exercises/two-fer/lib/example.dart

```diff
@@ -0,0 +1,3 @@
+String twofer([String name = "you"]) {
+  return "One for ${name is! String ? 'you' : name}, one for me.";
```

Comment:
The return statement can be replaced with the arrow syntax: ex: `String function(Parameter param) => expression;`